source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/display.tsx

```diff
@@ -13,6 +13,8 @@ import Owners from './owners';
 import Reviewers from './reviewers';
 import Settings from './settings';
 import Files from './files';
+import { IProvidedProps as ITimeProps } from '@lib/with-moment-timezone';
```

Comment:
do you need this?

Reply:
idk, my editor doesn't tell me when I have unused imports.

Reply:
looks like neither of these imports are needed though